src/schema-association-service-1espt.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License.
+*--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Utils } from 'vscode-uri';
+import * as azdev from 'azure-devops-node-api';
+import * as logger from './logger';
+import { AzureSession } from './typings/azure-account.api';
+import { VersionControlRecursionType } from 'azure-devops-node-api/interfaces/TfvcInterfaces';
+import * as fs from 'fs'
+import { Messages } from './messages';
+
+const milliseconds24hours = 86400000;
+
+export async function get1ESPTSchemaUriIfAvailable(azureDevOpsClient: azdev.WebApi, organizationName: string, session: AzureSession, context: vscode.ExtensionContext, lastUpdated1ESPTSchema: Date, seen1ESPTOrganizations: Set<string>): Promise<any> {
+    try {
+        if (session.userId.endsWith("@microsoft.com")) {
```

Comment:
Won't work for people with e.g. ntdev.microsoft.com emails - you should check both `@microsoft.com` and `.microsoft.com`

Reply:
Makes sense. Fixed it